template/apps/api/src/resources/subscription/actions/get-current.ts

```diff
@@ -0,0 +1,19 @@
+import { AppKoaContext, AppRouter } from 'types';
```

Comment:
do we really need subscription resource?
I think we can store subscription inside the user